public/scripts/models/project.js

```diff
@@ -24,6 +24,18 @@ var app = app || {};
 
 
   Project.initProjects = () => {
+    $.ajax({
+      url: '/github/user/repos'
+      ,method: 'GET'
```

Comment:
Looking ahead to 401, try to get in the habit of trailing commas in JS:

```javascript
$.ajax({
  url: '/github/user/repos',
  method: 'GET',
})
```

Some browsers don't like the extra trailing comma (particularly IE), but real projects will use a minifier (if not a transpiler like Babel or TypeScript).